src/components/ProfileCard/ProfileCard.tsx

```diff
@@ -45,7 +47,8 @@ const ProfileCard = ({ data, onClick, withStateStatus }: ProfileCardProps) => {
           <div className={styles.data}>
             <p className={styles.name}>{data.fullName}</p>
             <p className={`${getStackClasses} ${styles.wordWrap}`}>
-              {data.seniority} {data.position}&nbsp;Developer
+              {mapSeniorityLevel(data.seniority as SeniorityLevel)}{' '}
```

Comment:
**nitpick:** instead of casting here, let's allow strings in `mapSeniorityLevel`, so it's `mapSeniorityLevel(value: SeniorityLevel | string)`